react/link/style.css

```diff
@@ -0,0 +1,3 @@
+a.w-button {
```

Comment:
Can/should this be part of `/button/style.css`?
Since this is styling the button?